src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -247,11 +247,47 @@ public AzureAppConfigurationOptions Connect(Uri endpoint, TokenCredential creden
             }
```

Comment:
After parameter validation, can we just call the other overload that accept endpoint collection? It makes sure we have the consistent behavior.